import_export/admin.py

```diff
@@ -24,10 +24,30 @@
 
 
 class ImportExportMixinBase:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Store already set change_list_template to allow users to independently
+        # customize the change list object tools
+        if getattr(self, 'change_list_template', None):
```

Comment:
I think this might be able to be more concisely something like

```
self.base_change_list_template = getattr(self, 'change_list_template', 'admin/change_list.html') 
self.change_list_template = getattr(self, 'import_export_change_list_template', self.base_change_list_template)
```
Or something akin to that. But it's largely a matter of preference I guess :)

Reply:
And there you have that deep Django familiarity. Thanks for the detailed response. 

I believe an "or" might still have the desired effect but it isn't necessarily easier to understand. :)